src/devices/tuya.ts

```diff
@@ -6841,6 +6841,71 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
```

Comment:
This looks like a copy-paste of `MTG075-ZB-RL`, can you add `_TZE204_mtoaryre` to that definition instead of creating a new one?

Reply:
Sure, I'll do that. Thanks for all the work with the repo!